src/graphics/TFTDisplay.cpp

```diff
@@ -556,16 +556,12 @@ void TFTDisplay::sendCommand(uint8_t com)
         display(true);
         if (settingsMap[displayBacklight] > 0)
             digitalWrite(settingsMap[displayBacklight], TFT_BACKLIGHT_ON);
-#elif defined(ST7735_BL_V03)
```

Comment:
Do we need some testing on the Heltec trackers?

Reply:
That would be good. I'm just compiling for the 1.0 version. Results in a minute.

Reply:
Just noticed that after wake up from light sleep the BT is not turned back on. Not sure if this is a heltec only thing. But maybe unrelated to this PR, looks like problem in PowerFSM.

Reply:
I think I found the bug... it never worked :)

Reply:
Oh I was wrong... it did work but the change in #3596 / #3545 broke it.

Reply:
No, yes, no.... not working.... setBluetoothEnable() in main-esp32 is doing nothing when re-enabling.